### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by fulldecent** [fulldecent/FDChessboardView#8]
Should have 

```
lines = cd_stats[:total_lines_of_code].to_f
expectations = cd_stats[:total_test_expectations].to_f
 0.045 < (expectations / lines)
```

**Comment by fulldecent** [comment]
Add VERY simple tests just to confirm that the model API is working. Simply set the variable and then check that it has been set.

**Comment by fulldecent** [comment]
Fixed by 791d74bb9b1a53ef501b38257f270b469bd1c9fd
